### ISSUE TITLE
[FEATURE REQUEST] Select multiple series for blog

### ISSUE THREAD

**Issue by exaline-ru** [GinoPane/oc-blogtaxonomy-plugin#57]
Is it possible to select multiple series for one blog!? 

For example: I need to include an article in two different series:
"New Year's Tours" "One-Day Tours"

**Comment by GinoPane** [comment]
@exaline-ru no, currently it is not possible. Multiple series for one post will make them too similar to tags. Series are actually devoted to a list that is actively changing, like a TV show where new episodes are being added every week. Currently it is not possible to mark series as "active" or "ended" but it is a planned feature.

For me it sounds like tags would be more suitable for your task.

**Comment by exaline-ru** [comment]
In my request I mean: tags - as similar details of blogs, and the series - similar topics.

**Comment by LarBearrr** [comment]
@exaline-ru Use categories for this.

**Comment by exaline-ru** [comment]
But... all category content should not be included in the series.
